### DISCUSSION TITLE
Sharing Album or Folder ONLY

### DISCUSSION THREAD

**Discussion by ghost** [photoprism/photoprism#1718] [Q&A] [answered]
Hello,
Is it possible to share an album/folder with someone (using the built-in sharing function), but avoid them having the option of browsing my entire library?

I know I could move all my private pictures to "private", but I'd like to share a link of an album/folder for a few days to allow him to download it and keep my library as it currently is.

Thank you.

**Comment by lastzero** [comment, marked as answer]
Try using an incognito tab. When you open the link while you are still logged in as admin, you can obviously see your own library.

**Reply by ghost** [reply]
Sorry, forgot to mention I already tried that.

The whole library is still visible (labels, folders, albums etc)

**Reply by lastzero** [reply]
Are you running PhotoPrism in public mode?

**Reply by ghost** [reply]
No, I have set an initial admin password.
I am running photoprism as a docker on unRAID and for it to be in public_mode I would need to set "PHOTOPRISM_PUBLIC" to "true", which isn't the case.

**Reply by lastzero** [reply]
Can you test it again using a different browser? Accessing your entire library should obviously NOT be possible and we tested it many times.

**Reply by lastzero** [reply]
see https://github.com/photoprism/photoprism/issues/1421

**Reply by lastzero** [reply]
Also make sure to restart after changing the configuration 👍

**Reply by ghost** [reply]
For some reason I wasn't logged in as an admin and I didn't manage to remember my password.
Fortunately, for now my library is very small and I just deleted the database and the container, and reinstalled everything!

Now, logged is as admin I created a sharing link and it worked accordingly.

Thank you for all your time and patience. Sorry for the obvious mistake.
